src/java.base/share/classes/java/util/function/BiConsumer.java

```diff
@@ -72,4 +72,37 @@ default BiConsumer<T, U> andThen(BiConsumer<? super T, ? super U> after) {
             after.accept(l, r);
         };
     }
+
+    /**
+     * {@return a representation of the provided {@code uncaptured} lambda or method reference
+     * in the form of a {@code BiConsumer}}
```

Comment:
There's no need for the provided arguments to this and the other "of" methods to be a lambda or a method reference expression, although it is true all that the methods are useful especially for such cases.

Reply:
Also, is "capturing" the right term? IIRC, the JLS speaks about capture conversions for wildcards, but I'm not sure that the JLS concept also applies here.